### ISSUE TITLE
App.java:64-67: LCC metric has impediments (see puzzles...

### ISSUE THREAD

**Issue by 0pdd** [yegor256/jpeek#118]
The puzzle `9-e2b8b5ba` from #9 has to be resolved:

https://github.com/yegor256/jpeek/blob/b625770426cbfa670c987c135c0b3c98b7a24c8d/src/main/java/org/jpeek/App.java#L64-L67

The puzzle was created by Kapralov Sergey on 26-Jan-18. 

Estimate: 30 minutes, role: IMP.

If you have any technical questions, don't ask me, submit new tickets instead. The task will be "done" when the problem is fixed and the text of the puzzle is _removed_ from the source code. Here is more about [PDD](http://www.yegor256.com/2009/03/04/pdd.html) and [about me](http://www.yegor256.com/2017/04/05/pdd-in-action.html).

**Comment by 0crat** [comment]
@yegor256[/z](https://www.0crat.com/u/yegor256) please, pay attention to this issue

**Comment by 0crat** [comment]
Job [#118](https://github.com/yegor256/jpeek/issues/118) is now in scope, role is `DEV`

**Comment by 0crat** [comment]
The job [#118](https://github.com/yegor256/jpeek/issues/118) assigned to @llorllale[/z](https://www.0crat.com/u/llorllale). The budget is 30 minutes, see [§4](http://datum.zerocracy.com/pages/policy.html#4). Please, read [§8](http://datum.zerocracy.com/pages/policy.html#8) and [§9](http://datum.zerocracy.com/pages/policy.html#9). If the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html).

**Comment by llorllale** [comment]
blocked by #119

**Comment by llorllale** [comment]
@yegor256 this puzzle states it depends on another open puzzle (see #119). What can we do here while #119 isn't fixed?

**Comment by yegor256** [comment]
@llorllale just put it on hold, see http://datum.zerocracy.com/pages/policy.html#9

**Comment by llorllale** [comment]
@yegor256 then I'd have to assume the opportunity cost of having it take up one of the slots on my agenda as per http://datum.zerocracy.com/pages/policy.html#3.

Should this job have been put in `scope` before fixing its declared impediments?

**Comment by yegor256** [comment]
@llorllale you can reject it. Otherwise, you can try to move forward a bit, via PDD.

**Comment by llorllale** [comment]
@0crat refuse

**Comment by 0crat** [comment]
> @0crat refuse ([here](https://github.com/yegor256/jpeek/issues/118#issuecomment-364468170))

@llorllale The user @llorllale[/z](https://www.0crat.com/u/llorllale) resigned from [#118](https://github.com/yegor256/jpeek/issues/118), please stop working

**Comment by 0crat** [comment]
Tasks refusal is discouraged, see [§6](http://datum.zerocracy.com/pages/policy.html#6): -15 points just awarded to @llorllale[/z](https://www.0crat.com/u/llorllale), total is [+510](https://www.0crat.com/u/llorllale)

**Comment by 0crat** [comment]
The job [#118](https://github.com/yegor256/jpeek/issues/118) assigned to @nikialeksey[/z](https://www.0crat.com/u/nikialeksey), here is [why](https://www.0crat.com/footprint/C7JGJ00DP/440924558). The budget is 30 minutes, see [§4](http://datum.zerocracy.com/pages/policy.html#4). Please, read [§8](http://datum.zerocracy.com/pages/policy.html#8) and [§9](http://datum.zerocracy.com/pages/policy.html#9). If the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html).

**Comment by 0crat** [comment]
@nikialeksey[/z](https://www.0crat.com/u/nikialeksey) this job was assigned to you 5 days ago. It will be taken away from you soon, unless you close it, see [§8](http://datum.zerocracy.com/pages/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

**Comment by nikialeksey** [comment]
@0crat refuse

**Comment by 0crat** [comment]
> @0crat refuse ([here](https://github.com/yegor256/jpeek/issues/118#issuecomment-366536226))

@nikialeksey The user @nikialeksey[/z](https://www.0crat.com/u/nikialeksey) resigned from [#118](https://github.com/yegor256/jpeek/issues/118), please stop working

**Comment by 0crat** [comment]
Tasks refusal is discouraged, see [§6](http://datum.zerocracy.com/pages/policy.html#6): -15 points just awarded to @nikialeksey[/z](https://www.0crat.com/u/nikialeksey), total is [+60](https://www.0crat.com/u/nikialeksey)

**Comment by 0pdd** [comment]
@0pdd all puzzles are solved

**Comment by 0pdd** [comment]
@0pdd all puzzles are solved

**Comment by 0pdd** [comment]
@0pdd all puzzles are solved

**Comment by 0crat** [comment]
The job [#118](https://github.com/yegor256/jpeek/issues/118) assigned to @t-izbassar[/z](https://www.0crat.com/u/t-izbassar), here is [why](https://www.0crat.com/footprint/C7JGJ00DP/591356046). The budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4). Please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9). If the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html).

**Comment by t-izbassar** [comment]
@0crat waiting for #217 to be resolved as it is crucial part of introducing this metric.

**Comment by 0crat** [comment]
> @0crat waiting for #217 to be resolved as it is crucial part of introducing this metric. ([here](https://github.com/yegor256/jpeek/issues/118#issuecomment-369524603))

@t-izbassar The impediment for [#118](https://github.com/yegor256/jpeek/issues/118) was registered successfully by @t-izbassar[/z](https://www.0crat.com/u/t-izbassar)

**Comment by t-izbassar** [comment]
@0crat refuse

**Comment by 0crat** [comment]
> @0crat refuse ([here](https://github.com/yegor256/jpeek/issues/118#issuecomment-369858999))

@t-izbassar Job `gh:yegor256/jpeek#118` is not assigned, can't get performer

**Comment by t-izbassar** [comment]
@0crat status

**Comment by 0crat** [comment]
> @0crat status ([here](https://github.com/yegor256/jpeek/issues/118#issuecomment-369859750))

@t-izbassar This is what I know about this job, as in [§32](http://www.zerocracy.com/policy.html#32):

  * The job [#118](https://github.com/yegor256/jpeek/issues/118) is [in scope](http://www.zerocracy.com/policy.html#14) for 1mon
  * The role is `DEV`
  * The job is not assigned to anyone
  * The [budget](http://www.zerocracy.com/policy.html#4) is 30 minutes/points
  * These users are banned and won't be assigned:
    * @0pdd[/z](https://www.0crat.com/u/0pdd): This user reported the ticket
    * @llorllale[/z](https://www.0crat.com/u/llorllale): User was resigned from the ticket
    * @nikialeksey[/z](https://www.0crat.com/u/nikialeksey): User was resigned from the ticket
    * @t-izbassar[/z](https://www.0crat.com/u/t-izbassar): User was resigned from the ticket
  * Job [footprint](https://www.0crat.com/footprint/C7JGJ00DP?q=%7Bjob%3A%27gh%3Ayegor256%2Fjpeek%23118%27%7D) (restricted area)

**Comment by yegor256** [comment]
@0crat status

**Comment by 0crat** [comment]
> @0crat status ([here](https://github.com/yegor256/jpeek/issues/118#issuecomment-370418868))

@yegor256 This is what I know about this job, as in [§32](http://www.zerocracy.com/policy.html#32):

  * The job [#118](https://github.com/yegor256/jpeek/issues/118) is [in scope](http://www.zerocracy.com/policy.html#14) for 1mon
  * The role is `DEV`
  * The job is not assigned to anyone
  * The [budget](http://www.zerocracy.com/policy.html#4) is 30 minutes/points
  * These users are banned and won't be assigned:
    * @0pdd[/z](https://www.0crat.com/u/0pdd): This user reported the ticket
    * @llorllale[/z](https://www.0crat.com/u/llorllale): User was resigned from the ticket
    * @nikialeksey[/z](https://www.0crat.com/u/nikialeksey): User was resigned from the ticket
    * @t-izbassar[/z](https://www.0crat.com/u/t-izbassar): User was resigned from the ticket
  * Job [footprint](https://www.0crat.com/footprint/C7JGJ00DP?q=%7Bjob%3A%27gh%3Ayegor256%2Fjpeek%23118%27%7D) (restricted area)

**Comment by 0crat** [comment]
The job [#118](https://github.com/yegor256/jpeek/issues/118) assigned to @sharpler[/z](https://www.0crat.com/u/sharpler), here is [why](https://www.0crat.com/footprint/C7JGJ00DP/911130938); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html)

**Comment by 0crat** [comment]
@sharpler[/z](https://www.0crat.com/u/sharpler) this job was assigned to you 5days ago. It will be taken away from you soon, unless you close it, see [§8](http://www.zerocracy.com/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

**Comment by SharplEr** [comment]
@0crat waiting
For reviewer.

**Comment by 0crat** [comment]
> @0crat waiting For reviewer. ([here](https://github.com/yegor256/jpeek/issues/118#issuecomment-379464348))

@SharplEr The impediment for [#118](https://github.com/yegor256/jpeek/issues/118) was registered successfully by @sharpler[/z](https://www.0crat.com/u/sharpler)

**Comment by 0crat** [comment]
The job [#118](https://github.com/yegor256/jpeek/issues/118) assigned to @uurl[/z](https://www.0crat.com/u/uurl), here is [why](https://www.0crat.com/footprint/C7JGJ00DP/1350775815); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html)

**Comment by uurl** [comment]
@yegor256 ready for review

**Comment by uurl** [comment]
@0crat wait #232

**Comment by 0crat** [comment]
> @0crat wait #232 ([here](https://github.com/yegor256/jpeek/issues/118#issuecomment-389869866))

@uurl The impediment for [#118](https://github.com/yegor256/jpeek/issues/118) was registered successfully by @uurl[/z](https://www.0crat.com/u/uurl)

**Comment by uurl** [comment]
@0crat wait #239

**Comment by 0crat** [comment]
> @0crat wait #239 ([here](https://github.com/yegor256/jpeek/issues/118#issuecomment-390440578))

@uurl Job [#118](https://github.com/yegor256/jpeek/issues/118) is already on hold

**Comment by uurl** [comment]
@0crat status

**Comment by 0crat** [comment]
> @0crat status  ([here](https://github.com/yegor256/jpeek/issues/118#issuecomment-392087696))

@uurl This is what I know about this job, as in [§32](http://www.zerocracy.com/policy.html#32):

  * The job [#118](https://github.com/yegor256/jpeek/issues/118) is [in scope](http://www.zerocracy.com/policy.html#14) for 4mon
  * The role is `DEV`
  * The job is assigned to @uurl[/z](https://www.0crat.com/u/uurl) for [10days](http://www.zerocracy.com/policy.html#8)
  * There is a monetary reward attached
  * @uurl[/z](https://www.0crat.com/u/uurl) will get 50% of their hourly rate for this job
  * The job has an [impediment](http://www.zerocracy.com/policy.html#9)
  * The [budget](http://www.zerocracy.com/policy.html#4) is 30 minutes/points
  * These users are banned and won't be assigned:
    * @0pdd[/z](https://www.0crat.com/u/0pdd): This user reported the ticket
    * @llorllale[/z](https://www.0crat.com/u/llorllale): User was resigned from the ticket
    * @nikialeksey[/z](https://www.0crat.com/u/nikialeksey): User was resigned from the ticket
    * @t-izbassar[/z](https://www.0crat.com/u/t-izbassar): User was resigned from the ticket
    * @sharpler[/z](https://www.0crat.com/u/sharpler): User was resigned from the ticket
  * Job [footprint](https://www.0crat.com/footprint/C7JGJ00DP?q=%7Bjob%3A%27gh%3Ayegor256%2Fjpeek%23118%27%2Ctype%3A%7B%24not%3A%2F%5ENotify%2F%7D%7D) (restricted area)

**Comment by uurl** [comment]
@0crat wait #242

**Comment by 0crat** [comment]
> @0crat wait #242 ([here](https://github.com/yegor256/jpeek/issues/118#issuecomment-392118944))

@uurl Job [#118](https://github.com/yegor256/jpeek/issues/118) is already on hold

**Comment by uurl** [comment]
@yegor256 please close this with #242

**Comment by uurl** [comment]
@yegor256 please close this with #242

**Comment by 0pdd** [comment]
The puzzle `9-e2b8b5ba` has disappeared from the source code, that's why I closed this issue.

**Comment by 0pdd** [comment]
@0pdd 2 puzzles [#252](https://github.com/yegor256/jpeek/issues/252), [#253](https://github.com/yegor256/jpeek/issues/253) are still not solved.

**Comment by 0crat** [comment]
@elenavolokhova[/z](https://www.0crat.com/u/elenavolokhova) please review this job completed by @uurl[/z](https://www.0crat.com/u/uurl), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
The job [#118](https://github.com/yegor256/jpeek/issues/118) is now out of scope

**Comment by elenavolokhova** [comment]
@0crat quality good

**Comment by 0crat** [comment]
Order was finished, quality is "good": +35 point(s) just awarded to @uurl[/z](https://www.0crat.com/u/uurl)

**Comment by 0crat** [comment]
Quality review completed: +8 point(s) just awarded to @elenavolokhova[/z](https://www.0crat.com/u/elenavolokhova)

**Comment by 0pdd** [comment]
@0pdd 2 puzzles [#253](https://github.com/yegor256/jpeek/issues/253), [#356](https://github.com/yegor256/jpeek/issues/356) are still not solved; solved: [#252](https://github.com/yegor256/jpeek/issues/252).

**Comment by 0pdd** [comment]
@0pdd 2 puzzles [#252](https://github.com/yegor256/jpeek/issues/252), [#253](https://github.com/yegor256/jpeek/issues/253) are still not solved; solved: [#356](https://github.com/yegor256/jpeek/issues/356).

**Comment by 0pdd** [comment]
@0pdd the puzzle [#252](https://github.com/yegor256/jpeek/issues/252) is still not solved; solved: [#253](https://github.com/yegor256/jpeek/issues/253), [#356](https://github.com/yegor256/jpeek/issues/356).

**Comment by 0pdd** [comment]
@0pdd the puzzle [#514](https://github.com/cqfn/jpeek/issues/514) is still not solved.
